Compile errors in master

Pulling master (@ 80e83b8) results in compile errors:

```
$ dep ensure
$ go build ./...
# github.com/cloudflare/cloudflare-ingress-controller/pkg/controller
pkg/controller/controller.go:53:61: undefined: Config
pkg/controller/controller.go:174:34: undefined: IngressClassKey
pkg/controller/controller.go:176:50: undefined: IngressClassKey
pkg/controller/controller.go:179:39: undefined: IngressClassKey
pkg/controller/controller.go:547:40: undefined: IngressAnnotationLBPool
pkg/controller/controller.go:560:20: undefined: SecretLabelDomain
pkg/controller/controller.go:568:79: undefined: SecretLabelDomain
pkg/controller/controller.go:573:88: undefined: SecretLabelDomain
pkg/controller/controller.go:573:117: undefined: SecretName
pkg/controller/controller.go:574:70: undefined: SecretName
pkg/controller/controller.go:574:70: too many errors
```

The same errors are present when running `go install`:

```
$ go install github.com/cloudflare/cloudflare-ingress-controller/cmd/argot
# github.com/cloudflare/cloudflare-ingress-controller/pkg/controller
pkg/controller/controller.go:53:61: undefined: Config
pkg/controller/controller.go:174:34: undefined: IngressClassKey
pkg/controller/controller.go:176:50: undefined: IngressClassKey
pkg/controller/controller.go:179:39: undefined: IngressClassKey
pkg/controller/controller.go:547:40: undefined: IngressAnnotationLBPool
pkg/controller/controller.go:560:20: undefined: SecretLabelDomain
pkg/controller/controller.go:568:79: undefined: SecretLabelDomain
pkg/controller/controller.go:573:88: undefined: SecretLabelDomain
pkg/controller/controller.go:573:117: undefined: SecretName
pkg/controller/controller.go:574:70: undefined: SecretName
pkg/controller/controller.go:574:70: too many errors
```

--- a/pkg/controller/const.go
+++ b/pkg/controller/const.go
@@ -1,0 +1,10 @@
+package controller
+
+const (
+	MaxRetries                = 5
+	IngressClassKey           = "kubernetes.io/ingress.class"
+	IngressAnnotationLBPool   = "argo.cloudflare.com/lb-pool"
+	SecretLabelDomain         = "cloudflare-argo/domain"
+	SecretName                = "cloudflared-cert"
+	CloudflareArgoIngressType = "argo-tunnel"
+)

--- a/pkg/controller/controller.go
+++ b/pkg/controller/controller.go
@@ -50,6 +50,13 @@ type ArgoController struct {
 	tunnels map[string]tunnel.Tunnel
 }
 
+type Config struct {
+	IngressClass   string
+	KubeconfigPath string
+	Namespace      string
+	MaxRetries     int
+}
+
 func NewArgoController(client kubernetes.Interface, config *Config) *ArgoController {
 
 	informer, indexer, queue := createIngressInformer(client, config.IngressClass)

